bull/pdl-queue/consumer.js

```diff
@@ -112,27 +118,78 @@ async function uploadToIA(zip, metadata, byteLength, email, job) {
   );
 }
 
+async function uploadPdfToIA(uri, metadata, email, job) {
+  const requestURI = request(job.data.uri);
+  const bucketTitle = metadata.IAIdentifier;
+  const IAuri = `http://s3.us.archive.org/${bucketTitle}/${bucketTitle}.pdf`;
+  let headers = setHeaders(metadata, responseSize, metadata.title, pdf);
```

Comment:
1. "pdf" should be in string.
2. I think it's better you directly send the content type so that you don't need to do another check in the `setHeaders()` function.

Reply:
I've made the requested changes